lib/load_schema.js

```diff
@@ -0,0 +1,20 @@
+/**
+* load the shema user
+* @param {sting} schemaPath - the file path of the schema
+* @param {function} cb - callback with error and the schema object
+*/
+var fs = require('fs');
+
+module.exports = function (schemaPath, cb) {
+  fs.readFile(schemaPath, function (err, data) {
+    if (err) {
+      return cb('Error: sorry impossible to read the file at ' + schemaPath);
```

Comment:
The first is generated by `new Error(err)`, correct? What do you get if you just pass `err`?